qutebrowser/config/configdata.yml

```diff
@@ -643,6 +643,24 @@ completion.web_history_max_items:
 
     0: no history / -1: unlimited
 
+completion.delay:
+  default: 0
+  type:
+    name: Int
+    minval: 0
+    maxval: maxint64
+  desc: >-
+    Delay in ms before updating completions after typing a character
+
+completion.min_chars:
```

Comment:
I'm curious about the usefulness of this one ... the only completion I'm aware of right now that might have performance issues is `url`, and I imagine any url will be longer than a reasonable value of `min_chars`, so I don't see where this would take effect.